storage/remote/read.go

```diff
@@ -43,7 +43,7 @@ type querier struct {
 
 // Select implements storage.Querier and uses the given matchers to read series
 // sets from the Client.
-func (q *querier) Select(matchers ...*labels.Matcher) (storage.SeriesSet, error) {
+func (q *querier) Select(_ *storage.SelectParams, matchers ...*labels.Matcher) (storage.SeriesSet, error) {
```

Comment:
I think the params should be passed down to the proto we send for a read

Reply:
Now – or rather once we actually have someone who needs it?

Reply:
Noone has explicitly asked for it yet, but it is the plan of record from #2580

Reply:
Let's do it when we need it then.